ui/components/TracePage/TracePage.tsx

```diff
@@ -454,10 +462,20 @@ const CallChildren = ({
 const isObjectLike = (value: unknown): value is object =>
   value !== null && typeof value === "object";
 
-const getFirstDescendant = (value: unknown): unknown =>
-  isObjectLike(value) ? getFirstDescendant(Object.values(value)[0]) : value;
+const isStringArray = (value: unknown): value is unknown[] =>
```

Comment:
I missed this before, but since this only checks the first element and there could be other types in this array, can you change the name?

Reply:
Done